### DISCUSSION TITLE
Troubleshooting GitHub Enterprise Server

### DISCUSSION THREAD

**Discussion by topeogunleye** [community/community#124277] [Enterprise]

**Comment by queenofcorgis** [comment]
Hi @topeogunleye ,
The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was flagged and removed because it violated our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. Posting assignments will likely discourage engagement and responses.

If this content is posted again, your Community account will be subject to immediate and indefinite suspension.
For additional guidance, please review our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service).

Thank you!

**Reply by topeogunleye** [reply]
Hi @queenofcorgis,

Thank you for your message. I appreciate the effort you put into maintaining a positive and constructive environment within the GitHub Community on Discussions.

I'm very sorry for any inconvenience caused by my previous post. I will look over the Community Code of Conduct and GitHub's Terms of Service to make sure that I contribute in a respectful manner moving forward.

Have a great day! 😊

Best regards,
@topeogunleye
